src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/ValidationInputTagHelper.cs

```diff
@@ -141,6 +141,13 @@ private TagBuilder GetInputBuilder()
             }
 
             builder.AddCssClass(TagHelperConstants.NhsInput);
+
+            var modelStateErrors = ViewContext.ViewData.ModelState[For.Name]?.Errors;
```

Comment:
There are now two places the class is trying to find out whether or not a particular field is associated with any errors in the model state. Would it be possible to refactor this so the logic is not duplicated?

Reply:
👍